frontend-react/package.json

```diff
@@ -65,7 +68,7 @@
         "e2e": "cypress run",
         "e2e:open": "cypress open",
         "test": "react-scripts test",
-        "test:ci": "CI=true yarn --verbose test --coverage --verbose",
+        "test:ci": "CI=true yarn --verbose test --coverage --verbose --silent",
```

Comment:
```suggestion
        "test:ci": "CI=true yarn test --coverage --verbose --silent",
```

Duplicate flag

Reply:
my understanding is that those two flags actually do two different things - one is applied to the test command, and one is applied to the yarn command. it is dumb and maybe not totally necessary. do you like the output without the yarn flag better?

Reply:
Ah, you're right - disregard